microsoftgraph/microsoft-graph-explorer-v4
# Tells user they need to  be signed in to use PUT/POST/DELETE/PATCH

## Overview

Shows the user an error message and disables the Run Query button when the user selects either of POST|DELETE|PUT|PATCH while signed out.

### Demo

**GET**
![image](https://user-images.githubusercontent.com/12011447/104820488-f4530000-5845-11eb-8999-840b8681bd31.png)

**DELETE**
![image](https://user-images.githubusercontent.com/12011447/104820507-18aedc80-5846-11eb-9cd4-a9b6e7554e6d.png)

### Notes

The RUN QUERY button is disabled in addition to showing the error message so as to prevent users from making incorrect queries.

## Testing Instructions

* Sign out
* Change http method to either of PUT, PATCH, DELETE or POST
* Notice that you see an error message and the RUN QUERY button is disabled
* Sign In
* Change http method to either of PUT, PATCH, DELETE or POST
* Notice that you don't see an error message and RUN QUERY is active

Fixes #700 and closes #713
